tchannel/request.py

```diff
@@ -54,8 +64,11 @@ class Request(object):
     __slots__ = (
         'body',
         'headers',
+        'service',
```

Comment:
No. This is the service name the caller gave in the call. If the service is a proxy that will just forward requests, then the service name of the target service will be here.